Day-I-Social-Card/social-card/src/App.js

```diff
@@ -1,22 +1,22 @@
 import React from 'react';
 import './App.css';
+import HeaderContainer from './components/HeaderComponents/HeaderContainer'
```

Comment:
While these extra imports do not hurt anything, they are not necessary.  You only need to import the files you want to use i.e. HeaderContainer and CardContainer